.claude/scripts/ai-review/fetch_reviews.py

```
@@ -0,0 +1,254 @@
+        print(f"✅ 수집 완료: {len(filtered_comments)}개 댓글")
+        for bot in bots:
+            count = sum(1 for c in filtered_comments if c.bot_name == bot)
+            if count > 0:
+                print(f"  - {bot}: {count}개")
+
+        return filtered_comments
+
+    def _fetch_pr_issue_comments(self, pr_number: int) -> List[BotComment]:
+        """PR issue 댓글 수집 (일반 댓글)"""
+        try:
+            result = subprocess.run(
```

Ruff (0.14.3)

`subprocess` call: check for execution of untrusted input

(S603)